### PR TITLE
7 about me component

### LINKED ISSUE #7: Component: About me

### PR DESCRIPTION
**Pull request by adibNo9**
fix #7 close #7

### CODE REVIEW

File: libs/components/src/lib/about-me/index.tsx [hunk 1]
```diff
@@ -1,24 +1,37 @@
...
-    >
-      <h1>Welcome to AboutMe!</h1>
-    </div>
-  );
-}
+  const data = useData();
+  const renderBody = () => {
+    return (
+      <>
+        <TextField
```
Comment (ErfanSeidipoor) [inline libs/components/src/lib/about-me/index.tsx]: if you want to use shallow rendering it's better to use it for Header and TextArea separately.
```
renderHeader()
renderTextArea()
```

Comment (adibNo9) [reply libs/components/src/lib/about-me/index.tsx]: okay, it's done.

File: libs/components/src/lib/AboutMe/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,12 @@
+@import "../../../../theme/index";
+
+.container {
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
```
Comment (ErfanSeidipoor) [inline libs/components/src/lib/AboutMe/index.module.scss]: please use space from theme instead of this

Comment (adibNo9) [reply libs/components/src/lib/AboutMe/index.module.scss]: ok, it's done, spacing in theme has an issue, we talking about that in meeting.

**Pull request closed without merging**